nix/scripts/node_modules.sh

```diff
@@ -111,9 +111,14 @@ function replaceNodeModules() {
   echo "Copying node_modules from Nix store:" >&2
   echo " - ${src}" >&2
   copyNodeModules ${src} ${dst}
+  nodeify
```

Comment:
This will not work for CI builds, because the `nix/scripts/node_modules.sh` script is only used for shells, and is not used by the normal CI build process, since this is what happens in normal Nix build:
https://github.com/status-im/status-react/blob/c9c8263c9e35142693e2ebd2f640f340708296ee/nix/mobile/android/release.nix#L114-L117
A functional solution would involve changing `nix/deps/nodejs-patched/default.nix` instead.

Reply:
But I'd like to avoid adding any more special patching steps to our build process. So if possible a much better solution could maybe involve a meta package that does the shimming/polifill so it can be included as a normal dependency.

Reply:
For example, here is a package that provides various polifills for `react-native`:
https://github.com/acostalima/react-native-polyfill-globals#readme

Couldn't we do something similar for the `crypto` and `stream` package?